sdk/python/tests/test_historical_retrieval.py

```diff
@@ -281,7 +296,12 @@ def test_historical_features_from_parquet_sources():
 @pytest.mark.parametrize(
     "provider_type", ["local", "gcp"],
 )
-def test_historical_features_from_bigquery_sources(provider_type):
+@pytest.mark.parametrize(
+    "created_timestamp_column", ["created", ""],
```

Comment:
My only feedback is that these tests tend to run long, and will run even longer in the future the more we start to parameterize it. Would it be better to just have a single source (out of multiple) that doesnt have a created TS?